src/components/RequestDashboard/SettingsSection.jsx

```diff
@@ -0,0 +1,271 @@
+              });
+          });
+        })
+        .catch(e => {
+          console.log('Failed to retrieve list of QuestionnaireResponses');
+          console.log(e);
+        });
+    };
+
+  const resetPims =
+    ({ pimsUrl }) =>
+    () => {
+      let url = new URL(pimsUrl);
+      const resetUrl = url.origin + '/doctorOrders/api/deleteAll';
+      console.log('reset pims: ' + resetUrl);
+
+      fetch(resetUrl, {
+        method: 'DELETE'
+      })
+        .then(response => {
+          console.log('Reset pims: ');
+          console.log(response);
+        })
+        .catch(error => {
+          console.log('Reset pims error: ');
+          console.log(error);
+        });
+    };
+  const resetRemsAdmin =
+    ({ cdsUrl }) =>
+    () => {
+      let url = new URL(cdsUrl);
+      const resetUrl = url.origin + '/etasu/reset';
+
+      fetch(resetUrl, {
+        method: 'POST'
+      })
+        .then(response => {
+          console.log('Reset rems admin etasu: ');
+          console.log(response);
+        })
+        .catch(error => {
+          console.log('Reset rems admin error: ');
+          console.log(error);
+        });
+    };
+  const clearMedicationDispenses =
+    ({ ehrUrl, access_token }, consoleLog) =>
+    () => {
+      console.log('Clear MedicationDispenses from the EHR: ' + ehrUrl);
+      const client = FHIR.client({
+        serverUrl: ehrUrl,
+        ...(access_token ? { tokenResponse: access_token } : {})
+      });
+      client
+        .request('MedicationDispense', { flat: true })
+        .then(result => {
+          console.log(result);
+          result.forEach(resource => {
+            console.log(resource.id);
+            client
+              .delete('MedicationDispense/' + resource.id)
+              .then(result => {
+                consoleLog(
+                  'Successfully deleted MedicationDispense ' + resource.id + ' from EHR',
+                  types.info
+                );
+                console.log(result);
+              })
+              .catch(e => {
+                console.log('Failed to delete MedicationDispense ' + resource.id);
+                console.log(e);
+              });
+          });
+        })
+        .catch(e => {
+          console.log('Failed to retrieve list of MedicationDispense');
+          console.log(e);
+        });
+    };
+  const resetHeaderDefinitions = [
+    {
+      display: 'Clear In-Progress Forms',
+      key: 'clearQuestionnaireResponses',
+      reset: clearQuestionnaireResponses
+    },
+    {
+      display: 'Reset PIMS Database',
+      key: 'resetPims',
+      reset: resetPims
+    },
+    {
+      display: 'Reset REMS-Admin Database',
+      key: 'resetRemsAdmin',
+      reset: resetRemsAdmin
+    },
+    {
+      display: 'Clear EHR MedicationDispenses',
+      key: 'clearMedicationDispenses',
+      reset: clearMedicationDispenses
+    }
+  ];
+
+  let firstCheckbox = true;
+  let showBreak = true;
+  return (
+    <div>
```

Comment:
```suggestion
```

The only thing I'd suggest is removing the outermost div (which is what I did in my PR which touched SettingsBox). I figured to do that since we have so many nested divs in this repo, and a MUI 5 Box renders as a div by default.